privaterepo/ssh_user_list.go

```diff
@@ -0,0 +1,83 @@
+package privaterepo
+
+import (
+	"bytes"
+	"html/template"
+	"net/url"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/valuemodifier/path"
+)
+
+func ContentToSSHUserList(content string) (string, error) {
```

Comment:
This is a more opinionated way of extracting the SSH UserList from the default draughtsman config.

Reply:
It should ctx as a first arg.

Reply:
But that does not make sense here. There is no long running or asynchronous or synchronized logic being executed. It is only string to string transformation.